src/server/scripts/Custom/Solocraft.cpp

```diff
@@ -525,7 +525,7 @@ class solocraft_player_instance_handler : public PlayerScript {
             for (Group::member_citerator itr = groupMembers.begin(); itr != groupMembers.end(); ++itr)
             {
                 //Exclude player from the tally because the player is the one entering the dungeon
-                if (itr->guid != player->GetGUID())
+                if (itr->guid != player->GetGUID().GetCounter())
```

Comment:
This shouldn't be changed as it's not a SQL query

Reply:
Thanks. I corrected.